app/helpers/vm_helper/textual_summary.rb

```diff
@@ -406,7 +406,7 @@ def textual_placement_group
     h = {:label => _('Placement Group'),
          :icon  => "pficon-flavor",
          :value => (my_placement_group.nil? ? _("None") : my_placement_group.name)}
-    if placement_group && role_allows?(:feature => "placement_group_show")
+    if !my_placement_group.nil? && role_allows?(:feature => "placement_group_show_list")
```

Comment:
Actually, isn't the feature for `Display Individual Placement Groups` `placement_group_show`

Reply:
Yeah, you're right - here it will always show a single placement group (or 'none'). I'll verify locally before a push an update.